### PR TITLE
Updates for withdrawing functionality

### PR DESCRIPTION
**Pull request by josojo**
Incorporates changes introduced here: https://github.com/gnosis/dex-liquidity-provision/pull/279

--- There are some unrelated changes due to my prettifier. I am sorry about that ---

### CODE REVIEW

File: docs/tutorial-cmm.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Signing and posting multi-send transaction 0x09cb78a5a49f10305a2f108d45e8fa059e4
 Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
 ```
 
-Now, you should just follow the link, and sign and execute the transaction with the lower transaction-	ID first. It should look like this:
+Now, you should just follow the link, and sign and execute the transaction with the lower transaction-ID first. It should look like this:
```
Comment (fedgiac) [inline docs/tutorial-cmm.md]:
```suggestion
Now, you should just follow the link, and sign and execute the transaction with the lower transaction ID first. It should look like this:
```

File: docs/tutorial-cmm.md [hunk 2]
```diff
@@ -189,11 +189,10 @@ Withdrawing from Gnosis Protocol is always done in two steps: Requesting and Cla
 Withdrawal requests can be made with the following command:
 
 ```ssh
-npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --requestWithdraw --network=$MASTER_SAFE
+npx truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --network=$NETWORK_NAME
```
Comment (fedgiac) [inline docs/tutorial-cmm.md]:
```suggestion
npx truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated brackets]  --tokenIds=[comma-separated indices] --network=$NETWORK_NAME
```

File: docs/tutorial-cmm.md [hunk 3]
```diff
@@ -189,11 +189,10 @@ Withdrawing from Gnosis Protocol is always done in two steps: Requesting and Cla
...
 ```
 
-Here, the flag `--requestWithdraw` is the most important part, which tells the script to initiate the the withdraw request.
-The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]`. Do **not** write the square brackets into the command. Under normal usage, these are the exact same indices that were provided when setting up the CMM.
+The `tokenIndices` of the tokens, which you want to withdraw from the brackets, need to be specified via the flag `--tokenIds=[indices]`. Do **not** write the square brackets into the command. Under normal usage, these are the exact same indices that were provided when setting up the CMM.
```
Comment (fedgiac) [inline docs/tutorial-cmm.md]:
```suggestion
The indices of the tokens that you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[comma-separated indices]`. Do **not** write the square brackets into the command. Under normal usage, these are the exact same indices that were provided when setting up the CMM.
```

File: docs/tutorial-cmm.md [hunk 4]
```diff
@@ -189,11 +189,10 @@ Withdrawing from Gnosis Protocol is always done in two steps: Requesting and Cla
...
 The flag `--brackets=[comma separated brackets]` must contain the brackets that the CMM deployed. Please provide these addresses separated by commas, without any spaces, and without any square brackets.
```
Comment (fedgiac) [inline docs/tutorial-cmm.md]:
```suggestion
The flag `--brackets=[comma-separated brackets]` must contain the brackets that the CMM deployed. Please provide these addresses separated by commas, without any spaces, and without any square brackets.
```

File: docs/tutorial-cmm.md [hunk 5]
```diff
@@ -211,11 +210,13 @@ By running the withdraw request script, you have generated a transaction within
 The actual withdrawal transaction transferring the tokens back into the MASTER_SAFE from Gnosis Protocol can be initiated with the following command.
 
 ```ssh
-npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --withdraw --transferFundsToMaster --network=$MASTER_SAFE
+npx truffle exec scripts/claim_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --network=$NETWORK_NAME
```
Comment (fedgiac) [inline docs/tutorial-cmm.md]:
```suggestion
npx truffle exec scripts/claim_withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated brackets]  --tokenIds=[comma-separated indices] --network=$NETWORK_NAME
```

**Review by fedgiac — APPROVED** [review]
I confirm that the changes made in this PR reflect how the scripts should be used for withdrawing.
Reading this, I think it was a good decision to switch to `claim_withdraw` instead of `withdraw`.
I'm also adding some opinionated nitpick comments.

**Review by keikreutler — APPROVED** [review]
Small changes approved.

**Pull request merged**